app/Spago.hs

```diff
@@ -55,10 +55,10 @@ main = withUtf8 $ do
         -> Path.showPaths buildOptions whichPath
       Repl replPackageNames paths pursArgs depsOnly replPackage
         -> Spago.Build.repl replPackageNames paths pursArgs depsOnly replPackage
-      BundleApp modName tPath shouldBuild buildOptions
-        -> Spago.Build.bundleApp WithMain modName tPath shouldBuild buildOptions globalUsePsa
-      BundleModule modName tPath shouldBuild buildOptions
-        -> Spago.Build.bundleModule modName tPath shouldBuild buildOptions globalUsePsa
+      BundleApp modName tPath platform minify shouldBuild buildOptions
+        -> Run.withPursEnv globalUsePsa $ Spago.Build.bundleApp WithMain modName tPath platform minify shouldBuild buildOptions globalUsePsa
+      BundleModule modName tPath platform minify shouldBuild buildOptions
+        -> Run.withPursEnv globalUsePsa $ Spago.Build.bundleModule modName tPath platform minify shouldBuild buildOptions globalUsePsa
```

Comment:
At this point `bundleModule` is taking 7 arguments, which is a little much. We should make a new datatype `BundleOptions` and pass that one in

Reply:
Is WithoutMain synonymous with module bundling in v0.14? Or is it possible to bundle a module with main?

Reply:
It's not possible, see here:
https://github.com/purescript/spago/blob/3d1a5be3efe37e92e822d74056461c2f1f33099e/src/Spago/Build.hs#L403-L409

Reply:
Ok I simplified this and merged `bundleApp` and `bundleModule` functions. Moved the es bundling to `Build.hs`

Reply:
Also added the `BundleOptions` data type